src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -366,6 +366,12 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
       return
     }
 
+    const isFutureEffective = (
+      business.firstFiling.isFutureEffective === true &&
+      business.firstFiling.status !== FilingStatus.COMPLETED &&
```

Comment:
I see. Nice!

Reply:
I should have mentioned this before... this is the same check as here:
https://github.com/bcgov/business-filings-ui/blob/4f6698f3f4fb6e95fef6c4695ac3abb31b2afb21/src/components/Dashboard/FilingHistoryList/HeaderActions.vue#L123